Bio/Align/__init__.py

```diff
@@ -947,21 +947,27 @@ def __init__(self, target, query, path, score):
         self.path = path
 
     def __eq__(self, other):
+        """Check if two PairwiseAlignment have the same path."""
         return self.path == other.path
 
     def __ne__(self, other):
+        """Check if two PairwiseAlignment have different paths."""
         return self.path != other.path
 
     def __lt__(self, other):
+        """Check if _other_ has a smaller path."""
```

Comment:
Here and in the ones below, It's not so much about a path being smaller / bigger, it's more about the sort order (e.g should self come before other).